clang/test/CodeGenCXX/throw-expression-typeinfo-in-nonzero-default-address-space.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %clang_cc1 %s -triple spirv64-unknown-unknown -fsycl-is-device -emit-llvm -fcxx-exceptions -fexceptions -std=c++11 -o - | FileCheck %s
```

Comment:
Instead of duplicating all these files, could you add one additional RUN: line to the test you copied this from? No need to regenerate with update_cc_test_checks, just add a new check line with something like a CHECK-NONZERO-AS prefix?